shell/browser/browser_linux.cc

```diff
@@ -159,6 +160,13 @@ void Browser::ShowAboutPanel() {
     gtk_about_dialog_set_copyright(dialog, copyright.c_str());
   if (about_panel_options_.GetString("website", &website))
     gtk_about_dialog_set_website(dialog, website.c_str());
+  if (about_panel_options.Get("args", &args)) {
+    std::vector<const char*> cstrs{};
```

Comment:
The empty list initializer doesn't buy us anything here.

```suggestion
    std::vector<const char*> cstrs;
```